inc/views/base/forumdisplay/thread.twig

```diff
@@ -1,9 +1,10 @@
-{% set class %}{{ thread.bgcolor }} {{ thread.classType }}{% endset %}
+{% set class %}{{ thread.bgcolor }} forumdisplay_{% if thread.sticky %}sticky{% else %}regular{% endif %}{% endset %}
+{% set moved = (thread.moved[0] == 'moved') %}
 <tr class="inline_row">
     <td align="center" class="{{ class }}" width="2%"><span class="thread_status {{ thread.folder.value }}" title="{{ thread.folder.label }}">&nbsp;</span></td>
     <td align="center" class="{{ class }}" width="2%">
         {% if thread.icon and iconCache[thread.icon] %}
-            <img src="{{ iconCache[thread.icon].path }}" alt="{{ iconCache[thread.icon].name }}" title="{{ iconCache[thread.icon].name }}" />
+            <img src="{{ iconCache[thread.icon].path|replace('{theme}', theme.imgdir) }}" alt="{{ iconCache[thread.icon].name }}" title="{{ iconCache[thread.icon].name }}" />
```

Comment:
Doing this here seems a little bit odd to me, would make sense to do it in the backing code...